yetanotherco/aligned_layer
# fix: hotfix 500 explorer inactive operators bug

## Description

Hotfix on the explorer, when the operator is inactive we were not setting some variables in the map, so `/operators` route fails to render.

**Testing**
- Run a local devenet
- Register an operator
- Run the explorer
- Run script to fetch operators events: `make explorer_fetch_old_operators_strategies_restakes FROM_BLOCK=0`
- Eject the operator, for example using `config-operator-1.yaml`: 
```shell
cast send 0xf5059a5D33d5853360D16C683c16e67980206f36 --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 "ejectOperator(address,bytes)" 0x70997970C51812dc3A010C7d01b50e0d17dc79C8 0x00 
```
- Go to `/operators` it should not fail.

## Type of change

## Checklist
